src/main/java/trackerbot/utils/Ui.java

```diff
@@ -0,0 +1,87 @@
+package trackerbot.utils;
+
+import java.util.Scanner;
+
+/**
+ * Generates UI elements for TrackerBot.
+ *
+ * @author WZWren
+ * @version A-JavaDoc
+ */
+public class Ui {
+    /** Line separators for the console between paragraphs. **/
+    private static final String FORMAT_LINE =
```

Comment:
Good that you named your variables in accordance to the Java coding standards given in our course! Since these variables are constants, they are named in all uppercase using underscore to separate words